design/account.go

```diff
@@ -114,6 +114,19 @@ var _ = a.Resource("user", func() {
 var _ = a.Resource("users", func() {
 	a.BasePath("/users")
 
+	a.Action("verifyEmail", func() {
+		a.Routing(
+			a.GET("/verifyemail"),
+		)
+		a.Params(func() {
+			a.Param("code", d.String, "code")
+			a.Required("code")
+		})
+		a.Response(d.OK, emailApprovedData)
```

Comment:
We should return a redirect instead. User opens a link in a browser https://auth.openshift.io/api/users/verifyemail?code=... and after successful verification we redirect to "your email has been verified" page. Please create two issues for UX team: 
1) Email template
2) "your email has been verified" page (we should add the relative link to this page to our configuration and set it by default so we don't need to configure it for every cluster (preview/prod)). For now it can be _home page until we have this new page available.

Reply:
sure will make it `https://prod-preview.openshift.io/_gettingstarted` by default.

Reply:
@alexeykazakov there would be an error page and a success page ( as per UX's designs ) 
https://redhat.invisionapp.com/share/U2F2YVAT5

I was planning to have a UI redirect with a param `verified=true` and `verified=false` to the same URL. Based on the param the UI content in the page would be rendered.

What do you think ?
@alexeykazakov  @mindreeper2420 ( cant find Catherine here )

Reply:
The same URL with different params (verified=true/false) would simplify things on the back-end side. But not sure if it's OK for the front-end side ;)
cc: @catrobson

Reply:
@alexeykazakov @sbose78 That sounds good to me, I think this should be fine for the front-end side.

Reply:
Thanks @catrobson and @alexeykazakov !